support-frontend/assets/pages/paper-subscription-landing/components/content/subsCardTab.jsx

```diff
@@ -0,0 +1,95 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import Content from 'components/content/content';
+import Text from 'components/text/text';
+import GridImage from 'components/gridImage/gridImage';
+import { sendClickedEvent } from 'helpers/tracking/clickTracking';
+import { paperHasDeliveryEnabled } from 'helpers/subscriptions';
+import { Accordion, AccordionRow } from '@guardian/src-accordion';
+import { css } from '@emotion/core';
+import { neutral } from '@guardian/src-foundations/palette';
+import { textSans } from '@guardian/src-foundations/typography';
+import { space } from '@guardian/src-foundations';
+
+import { ContentHelpBlock, ContentForm, type ContentTabPropTypes } from './helpers';
+
+const accordionContainer = css`
+  background-color: ${neutral['97']};
+
+  p {
+    ${textSans.small()};
+    margin-bottom: ${space[3]}px;
+  }
+`;
+
+// ----- Content ----- //
+const SubsCardFaqBlock = () => (
+  <Content
+    border={paperHasDeliveryEnabled()}
+    image={<GridImage
+      gridId="paperVoucherFeature"
+      srcSizes={[750, 500, 140]}
+      sizes="(max-width: 740px) 100vw, 400px"
+      imgType="png"
+    />
+  }
+  >
+    <Text>
+      The Guardian subscription card can be used at any of the 40,000 shops and supermarkets with news kiosks
+      in the UK.
+    </Text>
+    <Text>
+      You can collect the newspaper from your local store or have your copies delivered by your newsagent.
+    </Text>
+    <Text>
+      <div css={accordionContainer}>
+        <Accordion>
+          <AccordionRow label="Collecting from multiple newsagents">
+            <p>
+              Present your card to a newsagent each time you collect the paper. The newsagent will scan your
+              card and will bee reimbursed for each transaction automatically.
+            </p>
+            <p><a href="https://imovo.org/guardianstorefinder">Find your nearest participating retailer</a></p>
```

Comment:
maybe put the url in a const above and use in both accordion rows?

Reply:
Thanks - shall do